airbyte-config/init/src/test/resources/invalid_definitions.yaml

```diff
@@ -0,0 +1,19 @@
+- name: Snowflake
```

Comment:
Was this intended to be part of that review? I do not see it being used anywhere here.

Reply:
nope. i was trying to write a test for `YamlSeedConfigPersistence` to check that the validation logic works, but unfortunately with the way it is structure that's really hard to do. forgot to remove this.